packages/cactus-plugin-ledger-connector-besu/src/test/typescript/integration/plugin-ledger-connector-besu/deploy-contract/get-record-locator.test.ts

```diff
@@ -265,34 +265,27 @@ describe("PluginLedgerConnectorBesu", () => {
     });
     expect(setNameOut).toBeTruthy();
 
-    try {
-      await connector.invokeContract({
-        contractName: HelloWorldContractJson.contractName,
-        contractAbi: HelloWorldContractJson.abi,
-        contractAddress,
-        invocationType: EthContractInvocationType.Send,
-        methodName: "setName",
-        params: [newName],
-        gas: 1000000,
-        signingCredential: {
-          ethAccount: testEthAccount.address,
-          secret: testEthAccount.privateKey,
-          type: Web3SigningCredentialType.PrivateKeyHex,
-        },
-        nonce: 1,
-      });
-      throw new Error("This operation should not have succeeded, but it did.");
-    } catch (ex: unknown) {
-      // 'Returned error: Nonce too low'
-      expect(ex).toHaveProperty(
-        "message",
-        expect.stringContaining("Nonce too low"),
-      );
-      expect(ex).toHaveProperty(
-        "stack",
-        expect.stringContaining("Nonce too low"),
-      );
-    }
+    await expect(connector.invokeContract({
```

Comment:
Haha, why do you have this in a non jest test? @ashnashahgrover 
Are you testing your code after the changes?

Reply:
My bad. I tested the other one (which was actually a tap test). Resolving